l10n_ir_base/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Iran - Base",
+    "version": "14.0.3.0.0",
+    "author": "fadoo"
```

Comment:
It is required to add the OCA here.
```suggestion
    "author": "fadoo, Odoo Community Association (OCA)"
```